### PR TITLE
Update .conversation--message text color

### PR DESCRIPTION
**Pull request by WevrLabs**
# Pull Request

## Description
**Update .conversation--message text color in the Conversations List for better hierarchy**

This is a minor change in the CSS of the Conversations List which changes the message text (.conversation--message) to a lighter gray color for better hierarchy.

### **Before:**
before

### **After:**
after

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by pranavrajs** [comment]
@nithindavid Can you please take a look at this?

**Comment by nithindavid** [comment]
Hi @WevrLabs, Thanks for the PR,
I would prefer our current style becuase

**Comment by WevrLabs** [comment]
> Hi @WevrLabs, Thanks for the PR,
> I would prefer our current style because though hierarchy wise the gray looks better but readability wise the current one is better. Let me know your thoughts.

Hi there :)
Yes, I guess the current color is better when it comes to readability, it's just that I noticed every time I visit the Dashboard I'd be looking for the new or the conversations with unread messages, but sometimes I find it hard to locate them, so my suggestion was to highlight and draw more attention to the unread ones by reserving the color darker for those ones and also making the contact's name bold as well, and leaving the message color for read ones and contact's name weight just as they were.
Screen Shot 2020-06-28 at 7 53 07 PM

Would love to hear your thoughts!

**Comment by nithindavid** [comment]
Hey @WevrLabs, This idea sounds good, We could do something like using the next possible `font-weight` for the user and the last message. Let me if this sounds good.

image

**Comment by WevrLabs** [comment]
Great!
Can't wait to see it live 🥳

**Comment by nithindavid** [comment]
@WevrLabs Let me know if you're taking this up.

**Comment by WevrLabs** [comment]
> @WevrLabs Let me know if you're taking this up.

Yes, I will make the changes and submit a PR for them.

File: app/javascript/dashboard/assets/scss/widgets/_conversation-card.scss [hunk 1]
```diff
@@ -62,7 +62,7 @@
   }
 
   .conversation--message {
-    color: $color-body;
+    color: $color-gray;
```
Comment (WevrLabs) [inline app/javascript/dashboard/assets/scss/widgets/_conversation-card.scss:65]: Will be reverted back to the original value.

File: app/javascript/dashboard/assets/scss/widgets/_conversation-card.scss [hunk 2]
```diff
@@ -62,7 +62,7 @@
   }
 
   .conversation--message {
-    color: $color-body;
+    color: $color-body; // reverted to source value
```
Comment (nithindavid) [inline app/javascript/dashboard/assets/scss/widgets/_conversation-card.scss:65]: Can you remove this comment ?

Comment (WevrLabs) [reply app/javascript/dashboard/assets/scss/widgets/_conversation-card.scss]: Removed.

**Pull request merged**